### PR TITLE
Fix compatibility with setuptools 39.

### LINKED ISSUE #8: Breaks with setuptools >= 39
I have `setuptools` 39.0.1 installed and get the following error:

```python
$ bin/checkversions buildout.cfg
# Checking buildout file buildout.cfg
Traceback (most recent call last):
  File "bin/checkversions", line 12, in <module>
    sys.exit(z3c.checkversions.main.main())
  File ".../z3c.checkversions-0.5-py3.6.egg/z3c/checkversions/main.py", line 92, in main
    checker.check(level=options.level)
  File ".../z3c.checkversions-0.5-py3.6.egg/z3c/checkversions/base.py", line 103, in check
    if _final_version(parsed_version) and not _final_version(dist.parsed_version):
  File ".../z3c.checkversions-0.5-py3.6.egg/z3c/checkversions/base.py", line 23, in _final_version
    for part in parsed_version:
TypeError: 'Version' object is not iterable
```

Maybe related to #6.

### PR DESCRIPTION
**Pull request by icemac**
Fixes #8.

### CODE REVIEW

File: src/z3c/checkversions/base.py [hunk 1]
```diff
@@ -16,14 +16,10 @@
 from setuptools import package_index
 
 
-_final_parts = '*final-', '*final'
 def _final_version(parsed_version):
     """Function copied from zc.buildout.easy_install._final_version
```
Comment (mgedmin) [inline src/z3c/checkversions/base.py]: Is the docstring still accurate now that you've replaced the implementation?

File: src/z3c/checkversions/base.py [hunk 2]
```diff
@@ -105,10 +101,8 @@ def check(self, level=0):
                     continue
                 # trunk the version tuple to the first `level` elements
                 # (and remove *final and pad both to the level length)
-                trunked_current = [x for x in parsed_version[:level]
-                                   if not x.startswith('*')]
-                trunked_candidate = [x for x in dist.parsed_version[:level]
-                                     if not x.startswith('*')]
+                trunked_current = parsed_version.base_version.split('.')[:level]
+                trunked_candidate = dist.parsed_version.base_version.split('.')[:level]
```
Comment (mgedmin) [inline src/z3c/checkversions/base.py]: The comment needs some updating -- the "remove *final" bit is no longer true.

**Comment by mgedmin** [comment]
Let's drop Python 2.6 from travis (and tox and setup.py and mention that in the changelog).

(We should also add 3.5 and 3.6 to travis and tox and setup.py and changelog.)

File: src/z3c/checkversions/base.py (hunk 1 above)
Comment (icemac) [reply src/z3c/checkversions/base.py]: Yes it is, I copied the implementation from the current version of `zc.buildout`.

File: src/z3c/checkversions/base.py (hunk 2 above)
Comment (icemac) [reply src/z3c/checkversions/base.py]: Good catch!

**Comment by icemac** [comment]
@mgedmin I dropped 2.6 and 3.3.

**Comment by icemac** [comment]
@mgedmin Could you create a release on PyPI or give me (`icemac`) the necessary rights to do so?

**Pull request merged**